packages/mattermost-redux/src/client/client4.ts

```diff
@@ -2037,6 +2037,13 @@ export default class Client4 {
         );
     };
 
+    getPostsByIds = (postIds: string[]) => {
+        return this.doFetch<StatusOK>(
```

Comment:
This return type seems incorrect since I'd assume it's returning an array of posts, not just a `StatusOK`